.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this issue and if possible provide links referring the changes.]
```

Comment:
`made in this Pull Request` would be more precise

Reply:
Yes done.